api/openapi.json

```diff
@@ -1,13 +1,9 @@
 {
   "openapi": "3.0.2",
   "info": {
-    "title": "osparc.io web API",
-    "description": "osparc-simcore public web API specifications",
-    "version": "0.4.0",
-    "x-logo": {
-      "url": "https://raw.githubusercontent.com/ITISFoundation/osparc-manual/b809d93619512eb60c827b7e769c6145758378d0/_media/osparc-logo.svg",
-      "altText": "osparc-simcore logo"
-    }
+    "title": "osparc.io web API (dev)",
+    "description": "osparc-simcore public API specifications",
+    "version": "0.4.5-dev"
```

Comment:
How would I know to what commit of the openapi-simcore vcs this OAS belongs? 
OK so it is here that we know that it is not released, right? i.e. `-dev`
Should we add some info int he version? SEE https://regex101.com/r/Ly7O1x/3/ 
https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string